dialplan/asterisk/extensions_lib_user.conf

```diff
@@ -53,6 +53,8 @@ same  =   n(compute_result),GotoIf($[${GOSUB_RETVAL} = 2]?BUSY,1)
 same  =   n,GotoIf($[${GOSUB_RETVAL} = 1]?CANCEL,1)
 
 same  =   n,CELGenUserEvent(WAZO_CALL_LOG_DESTINATION,type: user,uuid: ${WAZO_DST_UUID},name: ${WAZO_DST_NAME})
+same  =   n(screen_block),AGI(agi://${WAZO_AGID_IP}/screen_blocklist,${WAZO_DST_UUID})
```

Comment:
The blocklist should only be called if the call comes from an incall, we don't need to screen internal calls